tracee/consts.go

```diff
@@ -416,24 +422,12 @@ var EventsIDToName map[int32]string
 var EventsNameToID map[string]int32
 
 func init() {
-	len := len(EventsSyscalls) + len(EventsKprobes)
+	len := len(EventsIDToEvent)
 	EventsIDToName = make(map[int32]string, len)
 	EventsNameToID = make(map[string]int32, len)
-	for id, name := range EventsSyscalls {
-		EventsIDToName[id] = name
-		EventsNameToID[name] = id
-	}
-	for id, name := range EventsTracepoints {
-		EventsIDToName[id] = name
-		EventsNameToID[name] = id
-	}
-	for id, name := range EventsKprobes {
-		EventsIDToName[id] = name
-		EventsNameToID[name] = id
-	}
-	for id, name := range EventsKretprobes {
-		EventsIDToName[id] = name
-		EventsNameToID[name] = id
+	for id, event := range EventsIDToEvent {
+		EventsIDToName[id] = event.Name
+		EventsNameToID[event.Name] = event.ID
 	}
 }
 
```

Comment:
we can remove essentialEvents now, of if we want the index, just build it in init

Reply:
See my next PR